### PR TITLE
Updated release versions

### PR DESCRIPTION
**Pull request by kushagraThapar**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

File: sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
 
-### 2.8.0-beta.1 (Unreleased)
-
-#### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
-
+### 2.8.0 (2024-08-02)
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md:3]:
```suggestion
### 2.8.0 (2024-08-02)
```
### 2.8.0 (2024-02-08)

Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md:3]:
```suggestion
### 2.8.0 (2024-02-08)
```

File: sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,11 @@
 ## Release History
 
-### 4.26.0-beta.1 (Unreleased)
+### 4.26.0 (2024-08-02)
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md:3]:
```suggestion
### 4.26.0 (2024-02-08)
```

File: sdk/cosmos/azure-cosmos-spark_3-3_2-12/CHANGELOG.md [hunk 3]
```diff
@@ -1,13 +1,11 @@
 ## Release History
 
-### 4.26.0-beta.1 (Unreleased)
+### 4.26.0 (2024-08-02)
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-spark_3-3_2-12/CHANGELOG.md:3]:
```suggestion
### 4.26.0 (2024-02-08)
```

File: sdk/cosmos/azure-cosmos-spark_3-4_2-12/CHANGELOG.md [hunk 4]
```diff
@@ -1,13 +1,11 @@
 ## Release History
 
-### 4.26.0-beta.1 (Unreleased)
+### 4.26.0 (2024-08-02)
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-spark_3-4_2-12/CHANGELOG.md:3]:
```suggestion
### 4.26.0 (2024-02-08)
```

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 5]
```diff
@@ -1,15 +1,11 @@
 ## Release History
 
-### 4.55.0-beta.1 (Unreleased)
+### 4.55.0 (2024-08-02)
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:3]:
```suggestion
### 4.55.0 (2024-02-08)
```

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

**Pull request merged**